README.md

```diff
@@ -1,11 +0,0 @@
-[![Build Status](https://travis-ci.org/avast/datadog-scala-metrics.svg?branch=master)](https://travis-ci.org/avast/datadog-scala-metrics) [ ![Download](https://api.bintray.com/packages/avast/maven/datadog-scala-metrics/images/download.svg) ](https://bintray.com/avast/maven/datadog-scala-metrics/_latestVersion)
-
-# Datadog Scala Metrics
-
-Thin wrapper around datadog metrics that aims to provide idiomatic API for purely functional Scala.
-
-![Cats Friendly Badge](https://typelevel.org/cats/img/cats-badge-tiny.png) 
```

Comment:
I understand that it makes sense to remove quick start from readme, but I think you remove little too much.